code/modules/mob/living/simple_mob/subtypes/vore/morph/morph.dm

```diff
@@ -188,6 +189,14 @@
 		return
 	return ..()
 
+/mob/living/simple_mob/vore/hostile/morph/lay_down()
+	if(morphed)
+		var/temp_state = icon_state
+		..()
+		icon_state = temp_state
+	else
+		..()
+
```

Comment:
Why do this instead of just if(morphed) return?
Honest question: Are there cases where you need to 'get up' as a morph while morphed so you need it to modify if they're laying or not or what